### PR TITLE
import paths updated for zkSync scripts

### PR DESCRIPTION
**Pull request by 0xDEnYO**
# Which Jira task belongs to this PR?

# Why did I implement it this way?

# Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] This pull request is as small as possible and only tackles one problem
- [ ] I have added tests that cover the functionality / test the bug
- [ ] I have updated any required documentation

# Checklist for reviewer (DO NOT DEPLOY and contracts BEFORE CHECKING THIS!!!)

- [ ] I have checked that any arbitrary calls to external contracts are validated and or restricted
- [ ] I have checked that any privileged calls (i.e. storage modifications) are validated and or restricted
- [ ] I have ensured that any new contracts have had AT A MINIMUM 1 preliminary audit conducted on <date> by <company/auditor>

### CODE REVIEW

File: script/deploy/zksync/017_deploy_token_wrapper.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import { HardhatRuntimeEnvironment } from 'hardhat/types'
 import { DeployFunction } from 'hardhat-deploy/types'
 import { ethers, network } from 'hardhat'
-import { PeripheryRegistryFacet } from '../typechain'
+import { PeripheryRegistryFacet } from '../../../typechain'
```
[the remark this answers is not in the document]
Comment (0xDEnYO) [reply script/deploy/zksync/017_deploy_token_wrapper.ts:4]: outdated

File: script/deploy/zksync/007_deploy_fee_collector.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import { HardhatRuntimeEnvironment } from 'hardhat/types'
 import { DeployFunction } from 'hardhat-deploy/types'
 import { ethers, network } from 'hardhat'
-import { PeripheryRegistryFacet } from '../typechain'
+import { PeripheryRegistryFacet } from '../../../typechain'
```
[the remark this answers is not in the document]
Comment (0xDEnYO) [reply script/deploy/zksync/007_deploy_fee_collector.ts:4]: outdated

File: script/deploy/zksync/013_deploy_lifuel_fee_collector.ts [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import { HardhatRuntimeEnvironment } from 'hardhat/types'
 import { DeployFunction } from 'hardhat-deploy/types'
 import { ethers, network } from 'hardhat'
-import { PeripheryRegistryFacet } from '../typechain'
+import { PeripheryRegistryFacet } from '../../../typechain'
```
[the remark this answers is not in the document]
Comment (0xDEnYO) [reply script/deploy/zksync/013_deploy_lifuel_fee_collector.ts:4]: outdated

File: script/deploy/zksync/006_deploy_receiver.ts [hunk 4]
```diff
@@ -1,15 +1,15 @@
 import { HardhatRuntimeEnvironment } from 'hardhat/types'
 import { DeployFunction } from 'hardhat-deploy/types'
 import { ethers, network } from 'hardhat'
-import { Receiver, PeripheryRegistryFacet } from '../typechain'
+import { Receiver, PeripheryRegistryFacet } from '../../../typechain'
```
[the remark this answers is not in the document]
Comment (0xDEnYO) [reply script/deploy/zksync/006_deploy_receiver.ts:4]: outdated

File: script/utils/diamond.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { constants, Contract } from 'ethers'
 import { Fragment, FunctionFragment } from 'ethers/lib/utils'
 import { ethers } from 'hardhat'
-import { IDiamondCut, IDiamondLoupe } from '../typechain'
+import { IDiamondCut, IDiamondLoupe } from '../../typechain'
```
[the remark this answers is not in the document]
Comment (0xDEnYO) [reply script/utils/diamond.ts:4]: it is the correct path, I dont know why you keep highlighting these. We need to go two levels up to the root folder to be able to access the typechain folder.

File: script/deploy/zksync/008_deploy_service_fee_collector.ts [hunk 6]
```diff
@@ -1,13 +1,13 @@
 import { HardhatRuntimeEnvironment } from 'hardhat/types'
 import { DeployFunction } from 'hardhat-deploy/types'
 import { ethers, network } from 'hardhat'
-import { PeripheryRegistryFacet } from '../typechain'
+import { PeripheryRegistryFacet } from '../../../typechain'
```
[the remark this answers is not in the document]
Comment (0xDEnYO) [reply script/deploy/zksync/008_deploy_service_fee_collector.ts:4]: outdated

**Review by ezynda3 — APPROVED** [review]

**Review by ezynda3 — APPROVED** [review]

**Pull request merged**